README.md

```diff
@@ -245,9 +246,11 @@ Bridge worxlandroid:worxlandroidBridge:MyWorxBridge "MyWorx Bridge" [ webapiUser
 
 ### .items
 ```
+String Shaun							"Shaun [%s]"
```

Comment:
? I think we don't need this

Reply:
We do, to have current rights away in main screen. So you can see whether Shaun is home, mowing or any error even without moving into next menu. Even i'ts not related to binding, it's unsing this to update and as well a good example for a working rule, which is required for this.